Sources/sophiajs/src/sophia.ts

```diff
@@ -674,6 +676,35 @@
                 }
             };
         }
+
+        private attachOnLoadListeners() {
+            window.addEventListener('DOMContentLoaded', () => {
+                console.log('SophiaJS load event for frame with url ' + window.location.href);
+                const message = {
+                    ctx: 'sophia',
+                    name: 'pageLoadStateChange',
+                    payload: {
+                        state: 'domcontentloaded',
+                        url: window.location.href,
+                    },
+                };
+                window.webkit?.messageHandlers?.handler?.postMessage(message);
+            });
+
+            // On load.
```

Comment:
nit: no need for that comment - the code is clear

Reply:
👍